kassy

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+# kassy - A bash CLI tool for running kassy in docker
+# Written By: Matthew Hartstonge <matt@mykro.co.nz>
+
+# Soak in user params
+ARGS="$@"
+
+function usage() {
+  echo "usage: kassy [--all] [--fb] [--skype] [--slack] [--test]               "
```

Comment:
We could even have an optional API method for integration modules, where they expose a friendly name.